paywall/src/components/lock/Lock.js

```diff
@@ -68,6 +70,7 @@ Lock.propTypes = {
   purchaseKey: PropTypes.func.isRequired,
   config: UnlockPropTypes.configuration.isRequired,
   hideModal: PropTypes.func.isRequired,
+  showModal: PropTypes.func.isRequired,
```

Comment:
weird. Was it missing before? It looks like your changes do not use it?

Reply:
it was in fact missing. This is not part of the limited scope of this PR, want me to move it to a separate PR?